monitor-data/monitor-examples.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: 'Monitor examples'
+description: 'This page presents example monitor configurations for some common alerting use cases.'
+sidebarTitle: Examples
+tags:
+    ['axiom documentation', 'documentation', 'axiom', 'dashboards', 'monitors', 'notifiers', 'slack', 'email', 'pagerduty', 'webhook', 'opsgenie', 'discord', 'discord webhook']
+---
+
+## Notify on all occurrences of error
+
+To receive a notification on all occurrences of an error, create a match monitor where the filter conditions match the events reporting the error.
+
+To receive only certain attributes in the notification message, use the `project` operator.
+
+## Notify when error rate above threshold
+
+To receive a notification when the error rate exceeds a threshold, create a threshold monitor with an APL query that identifies the rate of error messages.
```

Comment:
Would it help build a relationship between Examples and Monitor types > Type pages if we made create a [threshold monitor](url) a link?

Reply:
Yes, really good point, thanks!